libs/components/forms/src/lib/modules/radio/radio-group.component.ts

```diff
@@ -248,6 +248,14 @@ export class SkyRadioGroupComponent
   @Input()
   public helpPopoverTitle: string | undefined;
 
+  /**
+   * A help key that identifies the global help content to display. When specified, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline) button is
+   * placed beside the  label. Clicking the button invokes global help as configured by the application.
```

Comment:
so this one should be "beside the heading" not label. Also looks like there may be an extra space in there.